doc/HOWTO

```diff
@@ -1,4 +1,4 @@
-[ Note: the installation instructions in this document are somewhat
+[ Note: The installation instructions in this document are somewhat
```

Comment:
I checked to see what's more common in our current documentation, and lowercase after `:` seems to be the most common:

```
$ find | grep '[1-9]\.xml$' | xargs grep -h ': [a-zA-Z]\+' | grep -v -e SPDX | grep -o ': [a-zA-Z]\+' | sort | uniq -c
      1 : In
      9 : NIS
      3 : can
      1 : file
      2 : group
      2 : if
      2 : invalid
      1 : it
     12 : max
      1 : maximum
      5 : no
      1 : pam
      1 : process
      1 : specified
      1 : strings
      1 : success
      1 : testuser
      1 : the
```

Or did you find a different pattern?

Also, at least in Spanish, the rule is that after colon you start with lowercase (<https://www.rae.es/dpd/dos%20puntos>).  I ignore if English has different rules.

Reply:
> This uppercase nonsense comes from marketing & AI. If you speak to people who have really studied the English language, they will tell you that after a colon, lowercase letters are used (with a few exceptions).

And from people with German background. :D